### DISCUSSION TITLE
postgrest heath check

### DISCUSSION THREAD

**Discussion by pallabganai** [PostgREST/postgrest#1967] [Q&A] [answered]
Which url i can use as livenessProbe & readinessProbe? I don't want to put extra burden on the database.

**Comment by wolfgangwalther** [comment]
Duplicate of #1891 and #1933.

**Comment by steve-chavez** [comment, marked as answer]
There's now proper health check endpoints on the latest pre-release: https://github.com/PostgREST/postgrest/releases/tag/v9.0.0.20220107
